File: app/src/main/java/com/kiwilss/lxkj/jetpacktest/app/MyApplication.java
package com.kiwilss.lxkj.jetpacktest.app;

import android.app.Application;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
            //依赖注入
            inject();

    }

    private void inject() {
        AppComponent component = DaggerAppComponent.builder()
                .appModule(new AppModule(this))
                .build();
        ComponentHolder.setAppComponent(component);
    }
}

File: zhihu/src/main/java/com/kiwilss/lxkj/zhihu/MyApplication.kt
package com.kiwilss.lxkj.zhihu

import android.app.Application
import com.kiwilss.lxkj.okhttp.OkhttpConfig


class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        //SkinManager.getInstance().init(this);
        OkhttpConfig.init(this)
    }
}